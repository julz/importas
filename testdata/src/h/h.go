package h

import (
	"github.com/onsi/gomega" // want `import "github.com/onsi/gomega" imported without alias but must be with alias "." according to config`
)

func foo() {
	gomega.Expect(nil).To(gomega.BeNil())
	gomega.
		Expect(true).To(gomega.BeTrue())
}
