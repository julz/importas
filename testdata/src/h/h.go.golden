package h

import (
	. "github.com/onsi/gomega" // want `import "github.com/onsi/gomega" imported without alias but must be with alias "." according to config`
)

func foo() {
	Expect(nil).To(BeNil())

	Expect(true).To(BeTrue())
}
